english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/duel/attack/attack.md

```diff
@@ -0,0 +1,8 @@
+You charge the roaring KitKat. His huge, wafer body sits atop two thin pretzel legs. He is top heavy. You decide to 
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.